projects/isy-angular-widgets/src/lib/wizard/components/wizard/wizard.component.spec.ts

```diff
@@ -364,10 +401,7 @@ describe('Integration Tests: WizardComponent with Mock Parent', () => {
 
   it('should have some classes on the back button', () => {
     const backButton = getNativeElementAsHTMLElement(backButtonDeclaration);
-    expect(backButton.className).toContain(CLASS_FLEX);
-    expect(backButton.className).toContain(CLASS_ALIGN_CENTER);
-    expect(backButton.className).toContain(CLASS_JUSTIFY_CENTER);
-    expect(backButton.className).toContain(CLASS_MR_2);
+    expect(backButton.className).toContain(CLASS_FLEX && CLASS_ALIGN_CENTER && CLASS_JUSTIFY_CENTER && CLASS_MR_2);
```

Comment:
This one is not working because in this case only the last expression (`CLASS_MR_2`) is evaluated. E.g. following expression would also pass the test:
`expect(backButton.className).toContain('bullshit' && CLASS_FLEX)`

Reply:
Done

Reply:
seems still to be open?